scripts/checkpatch.pl

```diff
@@ -5683,6 +5683,19 @@ sub process {
 			}
 		}
 
+# check for memcpy uses that should be memcpy_s
+		if ($line =~ /memcpy\s*\(.*\)\s*;/) {
```

Comment:
Is `$line` preprocessed to be real logical line? Because if it's just line from the code as it is, then it won't catch multiline calls like
```
memcpy(
        something
);
```

Reply:
Right as always ;) Changed regex.